src/main/java/uhtredragnarson/Deadline.java

```diff
@@ -0,0 +1,26 @@
+package uhtredragnarson;
+
+/**
+ * The Deadline class extends the Task class and represents a deadline that has a specific
+ * deadline timing.
+ */
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
While short, I didn't like the naming for the variable "by". Shouldn't both Deadline and Event have the same variable name (and also they do not consist of only timing) even though the tags "/at" and "/by" are different?